contxt/cli/utils.py

```diff
@@ -104,3 +104,12 @@ def __init__(self) -> None:
 
     def convert(self, *args, **kwargs) -> Any:
         return super().convert(*args, **kwargs).date()
+
+
+def str_to_bool(s):
+    if s == "True":
```

Comment:
consider something like:

```suggestion
    return s.lower() in ("true", "t", "1")
```

Reply:
this way the default is `False` and baker doesn't even have to specify a value for non-totalizer fields